boot/zephyr/Kconfig

```diff
@@ -1071,6 +1071,44 @@ config MCUBOOT_HW_DOWNGRADE_PREVENTION_COUNTER_LIMITED
 
 endchoice
 
+config MCUBOOT_UUID_VID
+	bool "Expect vendor unique identifier in image's TLV"
+	help
+	  Provide a vendor identification scheme to prevent processing images
+	  generated by a different vendor.
+
+config MCUBOOT_UUID_CID
+	bool "Expect image class unique identifier in image's TLV"
+	help
+	  Provide an image class identification scheme to prevent processing
+	  images for a different CPU or device produced by the same vendor.
+
+config MCUBOOT_UUID_VID_VALUE
+	string "Vendor name"
+	default ""
+	depends on MCUBOOT_UUID_VID || MCUBOOT_UUID_CID
+	help
+	  The vendor unique identifier.
+	  The following formats are supported:
+	   - Domain name (i.e. amce.corp) used to generate RFC 9562 UUID5
+	     identifier.
+	   - Raw UUID (i.e. 12345678-1234-5678-1234-567812345678)
+
+if MCUBOOT_UUID_CID
```

Comment:
can we have as many entries as images number allowed for given mcuboot assembly?